app/alarm/model/src/main/java/org/phoebus/applications/alarm/AlarmSystem.java

```diff
@@ -112,6 +112,9 @@
     /** Nag period in seconds */
     public static final long nag_period_ms;
 
+    /** Disable notify feature */
+    public static final String disable_notify_visible;
```

Comment:
disable_notify_visible should be a boolean flag, not a string with values 
true", "false".

Reply:
Field updated to boolean